lucene/ivy-versions.properties

```diff
@@ -24,7 +24,7 @@ com.fasterxml.jackson.core.version = 2.9.6
 /com.github.ben-manes.caffeine/caffeine = 2.4.0
 /com.github.virtuald/curvesapi = 1.04
 
-/com.google.guava/guava = 14.0.1
+/com.google.guava/guava = 25.1-jre
```

Comment:
Based on experience in KNOX-1611 found that 26.0-jre+ broke backwards compatibility for things Hadoop in integration tests required.